### DISCUSSION TITLE
ESP32-SSD1963-LVGL-Arduino

### DISCUSSION THREAD

**Discussion by ghost** [Bodmer/TFT_eSPI#2599] [Compatible displays and setup files]
I am creating a new project and I encountered this problem.

Board type - ESP32 WROOM
Display type - Riverdi 4,3" with SSD1963 parallel 8bit driver
Project created wit Square Line Studio.
Settings is:
Resolution: 480x272
Depth: 16 bit

The display connection is parallel 8 bits and the color depth is 16 bits. SSD1963 driver supports 18 bits. What would be the best solution? Can this problem be solved at all?

**Comment by Bodmer** [comment]
I am not sure what the question is, but maybe this helps:

The library supports 16 bit colour for the SSD1963.
Alathough the interface is only 8 bit, two bytes will be sent per pixel to produce the 16 bits for the colour.
